ExtIO_sddc/ExtIO_sddc.cpp

```diff
@@ -366,7 +367,7 @@ int64_t EXTIO_API SetHWLO64(int64_t LOfreq)
 	{
 		EXTIO_STATUS_CHANGE(pfnCallback, extHw_Changed_LO);
 	}
-
+        if (RadioHandler.GetmodeRF() == VHFMODE)
```

Comment:
TuneLO is able to handle all modes. I would prefer not checking here to keep consistent.

Reply:
Ok, You are right.